ui/page/root/home_page.go

```diff
@@ -170,6 +171,133 @@ func (hp *HomePage) OnNavigatedTo() {
 	hp.isBalanceHidden = hp.AssetsManager.IsTotalBalanceVisible()
 }
 
+// initDEX initializes a new dex client if dex is not ready.
+func (hp *HomePage) initDEX() {
+	if hp.AssetsManager.DexcReady() {
+		return // do nothing
+	}
+
+	go func() {
+		hp.AssetsManager.InitializeDEX(hp.dexCtx)
+
+		// If all went well, the dex client must be ready.
+		dexClient := hp.AssetsManager.DexClient()
+		if dexClient == nil {
+			return // nothing to do
+		}
+
+		// Wait until dex is ready
+		<-dexClient.Ready()
+
+		hasActiveOrders := dexClient.Active()
```

Comment:
dexClient.Active() should be dexClient.HasActiveOrders()

Reply:
This method comes from core.

Reply:
Interesting.